cli/agent.go

```diff
@@ -66,8 +66,9 @@ func (r *RootCmd) workspaceAgent() *clibase.Cmd {
 			// of zombie processes.
 			if reaper.IsInitProcess() && !noReap && isLinux {
 				logWriter := &lumberjack.Logger{
-					Filename: filepath.Join(logDir, "coder-agent-init.log"),
-					MaxSize:  5, // MB
+					Filename:   filepath.Join(logDir, "coder-agent-init.log"),
+					MaxSize:    5, // MB
+					MaxBackups: 1,
```

Comment:
I'm guessing this is to avoid gigabytes of logs in /tmp? Perhaps a comment for posterity and to avoid regressions?

Reply:
Yeah, it correctly rotates logs at 5mb but it leaves the old logs around forever without this.